### DISCUSSION TITLE
Afternoon

### DISCUSSION THREAD

**Discussion by scottjo** [lovata/oc-shopaholic-plugin#430] [General]
Just starting to look at Shopaholic for a solution for an online store etc. My question is "Is Shopaholic being actively maintained?" There seems to be a lot of issues but very little response etc. I don't want to spend a lot of time / money investing on something that isn't really supported.

Thoughts?

Jon

**Comment by lautsevich** [comment]
Hi @scottjo,

We appreciate your interest in Shopaholic products! Keep calm and use them confidently. We have recently released updates for plugins, and now most of them support all the latest features of October CMS, such as multisite for example. 

You can always contact us via the form on the October CMS website or in the following chats:
- https://t.me/oc_shopaholic_chat
- https://discord.gg/n7QXMJrj
